instrumentor/test/fixtures/is-switch/output.js

```diff
@@ -1,49 +1,12 @@
-'use strict';
+"use strict";
```

Comment:
does it matter if we use double vs single quotes? i noticed we use single quotes in other files.

Reply:
This doesn't matter.  It's an artifact of the fact `code.js` isn't strict and the output it inserting the declaration.  We don't care how Babel format quotes.